### DISCUSSION TITLE
how to upgrade offline installed milvus?

### DISCUSSION THREAD

**Discussion by dxsooo** [milvus-io/milvus#16098] [Q&A and General discussion]
https://milvus.io/cn/docs/v2.0.x/upgrade.md
here just talk about upgrade online milvus

**Comment by yhmo** [comment]
Not sure about your requirement. Could you provide more informations?

**Comment by LoveEachDay** [comment]
You can follow up the [offline installation procedures](https://milvus.io/cn/docs/v2.0.x/install_offline-helm.md) to save the newer version milvus image.

Then upload those images to your kubernetes cluster. 

Finally follow the [normal upgrade doc](https://milvus.io/cn/docs/v2.0.x/upgrade.md) to apply a milvus upgrade.
